### PR TITLE
Cinnabar/Zinc plus labels

### LINKED ISSUE #16: Refactor roads layer
Especially road text label sizes (smaller than 11 pt is probably too small), maybe add one way arrows?

### LINKED ISSUE #22: Buildings look bad in tilted 3d view
Using this style in Tangram ES with a tilted view the buildings get all smooched together. They need something to differentiate them, maybe add a vertical stroke? And/or change building exaggeration  (eg: Zinc doesn't exaggerate at all so even though it _would_ have the same problem, it doesn't because of the config)?

### LINKED ISSUE #25: Collision is disabled for some city labels
https://github.com/tangrams/cinnabar-style/blob/gh-pages/cinnabar-style.yaml#L4205

This has been observed to produce some overlapping labels in tangram-es:
screenshot_20170503-180040

### PR DESCRIPTION
**Pull request by sensescape**
- Adopt same **color themes** structure as Refill by creating a Cinnabar color theme and a Zinc color theme. _This retires Zinc as a stand-alone style._
- Adopt same **label steps** as Refill by adding 12 steps. Fixes https://github.com/tangrams/cinnabar-style/pull/42 and _retires `more-labels` and `no-labels` as official style variants._
- Internal refactors to show more map labels, roads (fixes #16, #20, #36, #37), buildings (fixes #22), water (less early small ponds and labels, intermittent water to dash), places (fixes #25), color globals cleanup, fix up the camera.

### CODE REVIEW

File: cinnabar-style.yaml [hunk 1]
```diff
@@ -1204,45 +987,108 @@ layers:
...
+                        order: global.feature_order_down_1
+                        width: [[13, 0.5px], [16, 1.5px], [18, 4m]]
                         join: round
+                intermittent:
+                    filter:
+                        any:
+                            - intermittent: [true]
+                            - kind: drain
+                    draw:
+                        lines:
```
Comment (nvkelso) [inline cinnabar-style.yaml:1056]: Let's make intermittent water lines dashed to match Bubble Wrap and Refill.

**Comment by nvkelso** [comment]
Can you also update the area filters for `water-area-labels:` to match Walkabout more (and for the water polygon area filters, too – they are slightly different).

File: cinnabar-style.yaml [hunk 2]
```diff
@@ -2412,57 +1848,30 @@ layers:
                         join: round
 
             in_park:
-                filter: { landuse_kind: [park,protected_area,nature_reserve,conservation,golf_course,garden,wood,farmland,farm] }
+                filter: { landuse_kind: [park, protected_area, nature_reserve, conservation, golf_course, garden, wood, farmland, farm, zoo] }
                 draw:
                     lines:
-                        #width: [[11, 0.5px], [12, 1.0px], [13, 1.2px], [14, 1.75px], [15, 2.5px], [16, 8m]]
-                        color: [[11, global.green1_r],[12, global.green1_r],[14, global.minor_road3]]
+                        color: [[11, global.green1_r], [12, global.green1_r], [14, global.minor_road3]]
```
Comment (nvkelso) [inline cinnabar-style.yaml]: Only use `minor_road3`, see unclassified road on Angle Island near SF which isn't visible.

File: cinnabar-style.yaml [hunk 3]
```diff
@@ -3122,191 +2645,7 @@ layers:
...
-                draw:
-                    text-blend-order:
-                        priority: 61
-                        visible: global.text_visible_path
-                        text_source: global.ux_language_text_source
-                        font:
-                            fill: global.text_fill
-                            size: 12px
-                            stroke: { color: global.text_stroke_road4, width: 4 }
-        track:
```
Comment (nvkelso) [inline cinnabar-style.yaml:3232]: I don't see track labels in the new version, please restore. See Coastal Trail near: 16/37.8333/-122.5379

File: index.html [hunk 4]
```diff
@@ -221,11 +221,24 @@
...
+        }
+
+        var tg_global = { sdk_mapzen_api_key: mapzen_api_key };
+
         var layer = Tangram.leafletLayer({
             // more: 'vector-tiles-M45zxG4'
             // default (some): 'vector-tiles-h6AbVZA'
             // no: 'vector-tiles-Bp6EQ60'
-            scene: { import: 'cinnabar-style.yaml', global: { sdk_mapzen_api_key: 'vector-tiles-h6AbVZA' } },
+            scene: { import: [tg_baseScene, 'themes/color-cinnabar.yaml'], global: tg_global },
```
Comment (nvkelso) [inline index.html]: Can just say:

```
            scene: { import: [tg_baseScene], global: tg_global },
```

Comment (nvkelso) [reply index.html]: Check against Refill for our pattern.

Comment (sensescape) [reply index.html]: https://github.com/tangrams/cinnabar-style/commit/5f31d87152a0e8fe07aec355eebe6af90718f12f

File: cinnabar-style.yaml (hunk 3 above)
Comment (sensescape) [reply cinnabar-style.yaml:3232]: https://github.com/tangrams/cinnabar-style/commit/5f31d87152a0e8fe07aec355eebe6af90718f12f

File: cinnabar-style.yaml (hunk 1 above)
Comment (sensescape) [reply cinnabar-style.yaml:1056]: https://github.com/tangrams/cinnabar-style/commit/5f31d87152a0e8fe07aec355eebe6af90718f12f

Comment (nvkelso) [reply cinnabar-style.yaml:1056]: Phoenix area: 16/33.5523/-112.0600

screen shot 2017-08-07 at 11 14 35

**Review by nvkelso — APPROVED** [review]
LGTM, thanks!

**Pull request merged**